src/components/WorldMap.js

```diff
@@ -115,7 +119,12 @@ export class WorldMap extends Component {
     this.state.map.remove();
   }
 
-  render() {
+  render(_, { map }) {
+    if (localStorage.getItem('geolocation') && map) {
+      const [lat, long] = getCoords();
+      map.setView([lat, long], 5);
```

Comment:
shouldn't this be done in `componentDidMount` ?

Reply:
Ah yes, fixed